### DISCUSSION TITLE
Request: Submit Fastfetch to the Windows Package Manager (winget-cli)

### DISCUSSION THREAD

**Discussion by iamhumanipromise** [fastfetch-cli/fastfetch#955] [Ideas]
Of course, neofetch is available. However, fastfetch is always superior! ;) 

Requesting it for the [winget-cli](https://github.com/microsoft/winget-cli) project. 

Here is an example of neofetch running in Windows: 

![image](https://github.com/fastfetch-cli/fastfetch/assets/105223045/ee688e73-cd59-4740-bb86-38370b6bd19e)

**Comment by CarterLi** [comment]
See: https://github.com/microsoft/winget-pkgs/pull/94697

Duplicate: #693 #703 #863

What is this neofetch? AFAIK, neofetch cann't be run in powershell and it can't generate this output either
